README.md

```diff
@@ -137,6 +137,24 @@ After apply the patch and rebuild spark, add following configuration in spark co
   spark.dynamicAllocation.enabled true
   ```
 
+## Deploy MapReduce Client
+
+1. Add client jar to Hadoop classpath, e.g., HADOOP_HOME/share/hadoop/mapreduce/
+
+The jar for MapReduce is located in <RSS_HOME>/jars/client/mr/rss-client-mr-XXXXX-shaded.jar
+
+2. Update MapReduce conf to enable Firestorm, the following demo is for local storage only, eg,
+
+   ```
+   -Dmapreduce.rss.coordinator.quorum=<coordinatorIp1>:19999,<coordinatorIp2>:19999
```

Comment:
we use this configuration to download jar, mapreduce.job.cache.files

Reply:
We use this configuration `mapreduce.application.classpath` to add ClassPath

Reply:
Ok